pycloudlib/azure/cloud.py

```diff
@@ -376,7 +376,7 @@ def _create_vm_parameters(self, name, image_id, nic_id, user_data):
         return vm_parameters
 
     def _create_virtual_machine(
-            self, image_id, nic_id, user_data, name, **kwargs
+        self, image_id, nic_id, user_data, name, vm_params=None
```

Comment:
@blackboxsw or @lucasmoura Would this break anything in ua-client? Are you guys overriding Azure vm_params via here (and line 404) anywhere?

Reply:
Safe from uaclient standpoint, integration tests  only currently call launch with image_id and user_data params and snapshot(instance=X).